pdc_client/plugins/release_variant.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2017 Red Hat
+# Licensed under The MIT License (MIT)
+# http://opensource.org/licenses/MIT
+#
+
+from __future__ import print_function
+
+import json
+try:
+    from collections import OrderedDict
+except ImportError:
+    # Python 2.6 needs this back-port
+    from ordereddict import OrderedDict
+
+from pdc_client.plugin_helpers import (PDCClientPlugin,
+                                       extract_arguments,
+                                       add_create_update_args)
+
+
+class ReleaseVariantPlugin(PDCClientPlugin):
+    def register(self):
+        self.set_command('release-variant')
+
+        # CRUD: CREATE
+        create_parser = self.add_action('create', help='create new release_variant')
+        self.add_release_variant_arguments(create_parser, required=True)
+        create_parser.set_defaults(func=self.release_variant_create)
+
+        # CRUD: READ one (info)
+        info_parser = self.add_action('info', help='display details of a release_variant')
+        info_parser.add_argument('release')
+        info_parser.add_argument('uid')
+        info_parser.set_defaults(func=self.release_variant_info, required=True)
+
+        # CRUD: READ many (list)
+        list_parser = self.add_action('list', help='list all release_variants')
+        self.add_release_variant_arguments(list_parser)
+        list_parser.set_defaults(func=self.release_variant_list)
+
+        # CRUD: UPDATE
+        update_parser = self.add_action('update', help='update an existing release_variant')
+        update_parser.add_argument('release')
+        update_parser.add_argument('uid')
+        self.add_release_variant_arguments(update_parser)
+        update_parser.set_defaults(func=self.release_variant_update)
+
+        # CRUD: DELETE
+        # not implemented
```

Comment:
Yes, another patch here adds the delete function.
Just remove the comment line from that commit.